plugins/inputs/openstack/openstack.go

```diff
@@ -7,6 +6,8 @@
 //
 // No aggregation is performed by the input plugin, instead queries to InfluxDB should
 // be used to gather global totals of things such as tag frequency.
+//
+//go:generate ../../../tools/readme_config_includer/generator
```

Comment:
@srebhan ran `make fmt` and this moved the go generate down. I thought it had to be the first line.

Reply:
Looks like this happened in 4 places